lib/matplotlib/widgets.py

```diff
@@ -2966,100 +3055,127 @@ def _release(self, event):
     def _onmove(self, event):
         """Motion notify event handler."""
 
-        state = self._state | self._default_state
+        state = self._state
+        rotate = ('rotate' in state and
+                  self._active_handle in self._corner_order)
+        eventpress = self._eventpress
+        # The calculations are done for rotation at zero: we apply inverse
+        # transformation to events except when we rotate and move
+        if not (self._active_handle == 'C' or rotate):
```

Comment:
```suggestion
        move = self._active_handle == 'C'
        if not move and not rotate:
```
for better readability.